domains/0xarchit.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "0xacrhit",
+    "mail": "0xarchit@proton.me"
+  },
+  "record": {
+    "TXT name": "0xarchit"
+    "TXT value": "zoho-verification=zb94087787.zmverify.zoho.in"
```

please read [the docs](https://docs.is-a.dev/).
```suggestion
    "TXT": [
      "0xarchit",
      "zoho-verification=zb94087787.zmverify.zoho.in"
    ]
```